DATM/AtmFieldUtils.F90

```diff
@@ -410,7 +410,7 @@ subroutine AtmBundleIntp(gcomp, exportState, externalClock, hour, rc)
     !                   real(AtmBundleFields(ii)%farrayPtr_fwd(ijloc(1),ijloc(2)),4)
     !call ESMF_LogWrite(trim(msgString), ESMF_LOGMSG_INFO)
 
-    if(dbug_flag > 0)then
+    if(dbug > 0)then
```

Comment:
I am curious, I feel dbug_flag is more meaningful then dbug, is there a particular reason we change it to "dbug"?

Reply:
I only did it only did it to make it more consistent w/ the other two caps (cice6,mom6), which read the config variable 'dbug_flag' and use it to set the value of the variable 'dbug'.

Reply:
OK, that makes sense. Thanks